scipy/sparse/csgraph/__init__.py

```diff
@@ -184,7 +186,8 @@
     breadth_first_tree, depth_first_tree, connected_components
 from ._min_spanning_tree import minimum_spanning_tree
 from ._flow import maximum_flow
-from ._matching import maximum_bipartite_matching
+from ._matching import maximum_bipartite_matching,\
```

Comment:
usually parentheses over slash for line continuations in SciPy I think

Reply:
I'm tempted to leave this one be, to remain consistent with the rest of the module, which currently looks like this:
```python
from ._shortest_path import shortest_path, floyd_warshall, dijkstra,\
    bellman_ford, johnson, NegativeCycleError
from ._traversal import breadth_first_order, depth_first_order, \
    breadth_first_tree, depth_first_tree, connected_components
from ._min_spanning_tree import minimum_spanning_tree
from ._flow import maximum_flow
from ._matching import maximum_bipartite_matching,\
    min_weight_full_bipartite_matching
from ._reordering import reverse_cuthill_mckee, structural_rank
from ._tools import construct_dist_matrix, reconstruct_path,\
    csgraph_from_dense, csgraph_to_dense, csgraph_masked_from_dense,\
    csgraph_from_masked, csgraph_to_masked
```
But just let me know if you'd rather we change all of them.

Reply:
Parentheses would indeed be better as backslashes are hard-coded line breaks; with parentheses editors and IDEs are happier. It would be great if you can touch all of them for conversion.

Reply:
Noted; done.